packages/guilded.js/lib/structures/Client.ts

```diff
@@ -51,6 +48,9 @@ export class Client extends (EventEmitter as unknown as new () => TypedEmitter<C
     users = new GlobalUserManager(this);
     bans = new GlobalMemberBanManager(this);
 
+    /** The user belonging to this bot */
+    user: ClientUser | null = null;
```

Comment:
Any reason this shouldn't be `user?: ClientUser;` to avoid the default null here.